bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FileInputImplV2.java

```diff
@@ -57,7 +57,7 @@ public String getDragDropText() {
     @Override
     public @NotNull Map<String, Object> getProperties() {
         Map<String, Object> customProperties = super.getProperties();
-        customProperties.put(ReservedProperties.PN_DRAG_DROP_TEXT, getDragDropText());
+        customProperties.put("fd:" + ReservedProperties.PN_DRAG_DROP_TEXT, getDragDropText());
```

Comment:
shouldn't it be FileInputImplV3?

Reply:
In v3 version, we have re-used the model of v2 as v3 revolves around the accessibility fixes in file-input component

Reply:
but then it would impact the v2 version also.

Reply:
@rismehta implemented the same!